app/helpers/rad_common/searchable_dropdown_helper.rb

```diff
@@ -0,0 +1,22 @@
+module RadCommon
+  MAX_DROPDOWN_SIZE = 300
+  module SearchableDropdownHelper
+    def searchable_scope_options(search_scope:, show_subtext: false, excluded_ids: nil)
```

Comment:
I moved this to a module to share this code, though not in love with this pattern if you have a better idea